src/MessageHandler/GithubImportCommandHandler.php

```diff
@@ -0,0 +1,24 @@
+<?php
+declare(strict_types=1);
+
+namespace App\MessageHandler;
+
+use App\Message\GithubImportCommand;
+use App\Message\ImportCommandCreator;
+use App\Services\GithubService;
+use Composer\Util\Git;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+class GithubImportCommandHandler implements MessageHandlerInterface {
+    private GithubService $githubService;
+
+    public function __construct(GithubService $githubService) {
+        $this->githubService = $githubService;
+    }
+    public function __invoke(GithubImportCommand $command) {
+        $this->githubService->importToDb($command->getOrgName());
+    }
+
+}
+
+

```

Comment:
```suggestion
class GithubImportCommandHandler implements MessageHandlerInterface 
{
    private GithubService $githubService;

    public function __construct(GithubService $githubService) 
    {
        $this->githubService = $githubService;
    }
    public function __invoke(GithubImportCommand $command): void 
    {
        $this->githubService->importToDb($command->getOrgName());
    }
}
```
sprawdź czy z tym voidem jest ok bo nie mogę podejrzeć do MessageHandlerInterface

Reply:
Jest ok